plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseTestHiveOnDataLake.java

```diff
@@ -1893,6 +1894,54 @@ public void testJsonS3SelectPushdownWithSpecialCharacters()
         }
     }
 
+    @Test
+    public void testS3SelectExperimentalPushdown()
+    {
+        // Demonstrate correctness issues which have resulted in CSV s3 select pushdown being put behind a separate "experimental" flag.
```

Comment:
```
```java
// Demonstrate correctness issues which have resulted in pushdown for TEXTFILE
// using CSV support in S3 Select being put behind a separate "experimental" flag.